src/Daemon/main.cpp

```diff
@@ -96,6 +98,9 @@ int main(int argc, char *argv[])
       case 'p':
 	pid_file = std::string(optarg);
 	break;
+      case 'P':
+  USBGUARD_LOG(Warning) << "PERMISSIONS CHECK ON POLICY FILE ARE TURNED OFF!", check_permissions = false;
```

Comment:
Please don't use the ',' operator to separate the expressions. Just write it on 2 lines. IMHO it's more readable that way.